nym-node/src/config/template.rs

```diff
@@ -237,6 +240,10 @@ upstream_exit_policy_url = '{{ exit_gateway.upstream_exit_policy_url }}'
 # derived shared keys, available client bandwidths and wireguard peers.
 clients_storage = '{{ exit_gateway.storage_paths.clients_storage }}'
 
+#Path to sqlite database containing all persistent stats data.
+stats_storage = '{{ exit_gateway.storage_paths.stats_storage }}'
```

Comment:
why does `nym-node` have 2 separate storages for exit and entry gateway? Just put it in one location and share it between "modes" since it's the same underlying gateway

Reply:
Nothing to fix, following internal discussion